tests/src/test/scala/whisk/core/controller/test/WebActionsApiTests.scala

```diff
@@ -890,7 +998,7 @@ trait WebActionsApiTests extends ControllerTestCommon with BeforeAndAfterEach wi
                         invocationsAllowed += 1
                         actionResult = Some(JsObject(
                             "headers" -> JsObject(
-                                "content-type" -> "xyz/bar".toJson),
+                                "content-type" -> "xyzbar".toJson),
```

Comment:
> another way would be to create list of custom media types and content types for what you need.

@dubeejw How would that work from a web action? The number of media types that can be used in the Content-Type header is essentially infinite.

For example, I may want to create `application/vnd.akrabat.bookshelf.v1+json` for a given project. This clearly won't be defined by OpenWhisk and it shouldn't prevent me from sending a content-type header with that media type.